README.md

```diff
@@ -25,6 +29,21 @@ call.
 
 The only required key is `sqlalchemy_url`, which provides the database URL.
 
+#### `asyncio` support using [`asyncpg`]
+
+SQLAlchemy `>= 1.4` supports `asyncio`.
+To enable `asyncio` support, install `asyncpg` extra to install [`asyncpg`]:
+
+```bash
+pip install fastapi-sqla[asyncpg]
```

Comment:
Do we not expect people to maintain `sqlalchemy` dependency themselves? If we do, should we encourage them to install the asyncpg version of sqlalchemy instead of having extras here?

Reply:
I do not see any sqla extras for asyncpg in https://github.com/sqlalchemy/sqlalchemy/blob/master/setup.py

```suggestion
pip install asyncpg
```
Should we just do this?

Reply:
yeah, it's easier IMO